server/src/main/kotlin/org/jetbrains/bsp/bazel/server/bep/BepServer.kt

```diff
@@ -126,6 +126,11 @@ class BepServer(
         else -> TestStatus.FAILED
       }
 
+      val coverageReportUri = testResult.testActionOutputList.find { it.name == "test.lcov" }?.uri
+      if (coverageReportUri != null) {
+        bspClientTestNotifier.sendCoverageReport(parentId, coverageReportUri)
+      }
+
```

Comment:
the progress notification is going to be sent before the test even started ;)

Reply:
Since this report is associated with the target, I had put it as a progress notification on the target's test task (sent on line 107), instead of as progress on an individual test case .  Let me know what you think.

Reply:
ahh, yes, my bad. Then it looks good :)